packages/ffe-datepicker-react/src/datepicker/Datepicker.spec.js

```diff
@@ -6,6 +6,7 @@ import Calendar from '../calendar';
 import Datepicker from './Datepicker';
 import i18n from '../i18n/i18n';
 import ErrorTypes from '../datelogic/error-types';
+import { KeyCodes } from '../../../ffe-account-selector-react/src/util/types';
```

Comment:
Cheeky ;) ~~I prefer duplication over reuse in this case. Either that or move `types` to a shared location somehow, but that seems out of scope.~~

There's already one here in [ffe-datepicker-react](https://github.com/SpareBank1/designsystem/blob/f0db1201b8cccab09967ec2aae3782a1d2ae3add/packages/ffe-datepicker-react/src/util/keyCode.js#L3)

Reply:
Oh my.. I intended to use the same constants as defined within the package, but as I wrote "keyCodes" instead of "keyCode" IntelliJ was being overly helpful in importing this for me :)
Fixed in the most recent commit, which is an amend commit of the previous squashed commits.